### DISCUSSION TITLE
Hellldivers Stratagem Hero Funbox

### DISCUSSION THREAD

**Discussion by rushp7** [monkeytypegame/monkeytype#5655] [Ideas]
I had an idea to create a PR for "Stratagem Hero" funbox similar to [https://combustibletoast.github.io/](url) which is based on the game Helldiver's 2. Wondering if this is something feasible and may be added to a future release before I attempt to implement it

**Comment by fehmer** [comment]
like funbox arrows?
![image](https://github.com/user-attachments/assets/9252a1d9-2e94-4686-803b-420e0069461c)

**Reply by rushp7** [reply]
Yes, pretty much it. But specific for combinations to practice for the game. I haven't looked into it in depth, but adding the sfx similar to one in the game could also be a possibility, or maybe using one of the existing sounds
